lib/components/artist/articles/index.js

```diff
@@ -16,7 +16,10 @@ class Articles extends React.Component {
     return (
       <View>
         <SerifText style={styles.heading}>Featured Articles</SerifText>
-        <ScrollView horizontal={true} style={{ overflow: 'visible', marginBottom:40 }}>
+        <ScrollView horizontal={true}
+                    showsHorizontalScrollIndicator={false}
+                    scrollsToTop={false}
```

Comment:
whoooops nevermind lol, saw the part in the artist.js that sets scroll to top to true :).